packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -356,17 +363,36 @@ export function generateGrid(
                         sortable: !sortFields.includes(column.name) ? `false` : undefined,
                         valueGetter: column.valueGetter,
                         valueOptions: column.valueOptions,
-                        width: column.width ? String(column.width) : "150",
                         renderCell: column.renderCell,
-                    }),
-                )
+                        width: typeof column.width === "undefined" ? undefined : String(column.width),
+                        flex: typeof column.flex === "undefined" ? undefined : String(column.flex),
```

Comment:
Do we need these String casts here?

Reply:
Good point, it's not required, fixed in 9884ec4802a0b945c8d25b2c1dd132287846b4c7